x-pack/plugins/alerting/server/authorization/audit_logger.ts

```diff
@@ -17,7 +17,7 @@ export enum AuthorizationResult {
   Authorized = 'Authorized',
 }
 
-export class AlertsAuthorizationAuditLogger {
```

Comment:
@legrego I have added the `release_note:breaking` label and added a line in the PR description about the change to audit events.

Reply:
Thanks, @ymao1! I think it would be beneficial to our users if we also gave some examples of the changes made to the audit log. For example, mentioning that the audit event types have changed (`alerts_authorization_failure` -> `alerting_authorization_failure`, etc.)

Reply:
@legrego PR description updated!